nss/Survey_NSS/02_Table of Means NSS 70.do

```diff
@@ -67,35 +68,42 @@ bysort urban: sum legal_own [aw = hhwgt],de
 ****************************************************************************
 use "${r_input}\Visit 1_Block 14",clear
 
-*debt
-gen debt = b14_q17
+*debt -- paper says liabilities on date of survey. pp16: "Panel A of Figure 2 reports the average allocation of liabilities across all households that carry a positive amount of debt at the date of the survey."
+gen debt = b14_q17 // 17 is on 30.06.2012
 sum debt,de
 
 *in paper: keep households with total liability >0 
 gen temp_debt =  (b14_q1 == "99")*debt
-egen total_debt = sum(temp_debt), by(HHID)
+egen double total_debt = sum(temp_debt), by(HHID)
 drop temp_debt
 
 gen temp_debt =  (b14_q1 != "99")*debt
-egen total_debt_m = sum(temp_debt),by(HHID)
+egen double total_debt_m = sum(temp_debt),by(HHID)
 drop temp_debt
 
```

Comment:
```
gen temp_debt_todt = (b14_q1 == "99")*b14_q16 //to date of survey the total debt. 
egen double total_debt_todt = sum(temp_debt_todt ), by(HHID)
drop temp_debt_todt
```

Reply:
Code the liability on the date of the survey to later use as a filter. pp16: "Panel A of Figure 2 reports the average allocation of liabilities across all households that carry a positive amount of debt at the date of the survey."